src/containers/my-courses/courses-filters-drawer/CoursesFiltersDrawer.tsx

```diff
@@ -28,6 +29,7 @@ interface CoursesFiltersDrawerProps {
   onClose: () => void
   isOpen: boolean
   handleReset: () => void
+  deviceFields: ReactNode | null
```

Comment:
maybe you can make it not required instead of null?

Reply:
made optional